<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_brake_distance</class>
 <widget class="QDialog" name="settings_brake_distance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure auto deceleration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="terminal.qrc">
    <normaloff>:/images/brake_setings.png</normaloff>:/images/brake_setings.png</iconset>
  </property>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>-10</y>
     <width>411</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="terminal.qrc">:/images/brake_distance.png</pixmap>
   </property>
  </widget>
  <widget class="QCheckBox" name="frontL">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="frontR">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="rearL">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="rearR">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="distance_for_brake">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>150</y>
     <width>43</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>456</x>
     <y>150</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>71</y>
     <width>201</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Enable auto deceleration</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="on_off">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Time before deceleration</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="time_before_deceleration">
      <property name="maximum">
       <number>20000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Deceleration time</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="deceleration_time">
      <property name="maximum">
       <number>20000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Deceleration force</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="deceleration_force">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>distance to braking</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="auto_deceleration_state">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto deceleration on</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="deceleration_done">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Deceleration done</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>apply</zorder>
  <zorder>label</zorder>
  <zorder>frontL</zorder>
  <zorder>frontR</zorder>
  <zorder>rearL</zorder>
  <zorder>rearR</zorder>
  <zorder>distance_for_brake</zorder>
  <zorder>label_9</zorder>
  <zorder>label_2</zorder>
  <zorder>auto_deceleration_state</zorder>
  <zorder>deceleration_done</zorder>
 </widget>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
